Orthtree/include/CGAL/Orthtree.h

```diff
@@ -43,338 +46,258 @@
 namespace CGAL {
 
 /*!
-  \ingroup PkgOrthtreeClasses
+  \ingroup PkgOrthtreeRef
 
-  \brief A data structure using an axis-aligned hybercubic
-  decomposition of dD space for efficient point access and
-  computations.
+  \brief A data structure using an axis-aligned hyperrectangle
+  decomposition of dD space for efficient access and
+  computation.
 
-  \details It builds a hierarchy of nodes which subdivide the space
-  based on a collection of points.  Each node represents an
-  axis-aligned hypercubic region of space.  A node contains the range
-  of points that are present in the region it defines, and it may
+  \details It builds a hierarchy of nodes which subdivides the space.
+  Each node represents an axis-aligned hyperrectangle region of space.
+  The contents of nodes depend on the traits class, non-leaf nodes also
   contain \f$2^{dim}\f$ other nodes which further subdivide the
   region.
 
   \sa `CGAL::Quadtree`
   \sa `CGAL::Octree`
 
-  \tparam Traits_ must be a model of `OrthtreeTraits`
-  \tparam PointRange_ must be a model of range whose value type is the key type of `PointMap_`
-  \tparam PointMap_ must be a model of `ReadablePropertyMap` whose value type is `Traits_::Point_d`
+  \tparam GeomTraits must be a model of `OrthtreeTraits`
  */
-template<typename Traits_, typename PointRange_,
-         typename PointMap_ = Identity_property_map<typename Traits_::Point_d> >
-class Orthtree
-{
+template <typename GeomTraits>
```

Comment:
Did we want to rename `GeomTraits` to `OrthtreeTraits`, similar to how it is done for [`AABB_tree`](https://doc.cgal.org/latest/AABB_tree/classCGAL_1_1AABB__tree.html)?

Reply:
Sounds good to me, GeomTraits is very generic.